src/main/java/backend/Core.java

```diff
@@ -67,13 +68,13 @@ public String currentStep() {
    * @return the step id just added
```

Comment:
Should probs update this docstring to say `the new current step` or something like that

Reply:
Sounds good!